developing-with-nats/sending/replyto.md

```diff
@@ -22,7 +22,7 @@ if err != nil {
 
 // Send the request.
 // If processing is synchronous, use Request() which returns the response message.
-if err := nc.PublishRequest("time", uniqueReplyTo, nil); err != nil {
+if err := nc.PublishRequest(uniqueReplyTo, "time", nil); err != nil {
```

Comment:
This is wrong. The original code is correct: you send the request to the public subject "time" and set the reply subject to `uniqueReplyTo` (second parameter). This will set the published message subject to "time" and reply subject to `uniqueReplyTo`